.github/workflows/release.yaml

```diff
@@ -323,13 +323,21 @@ jobs:
       - uses: actions/setup-node@v2.5.1
         with:
           node-version: "16"
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.17"
+      - run: go install sigs.k8s.io/bom/cmd/bom@v0.2.0
+      - run: go install github.com/spdx/spdx-sbom-generator/cmd/generator@v0.0.13
```

Comment:
interesting.. I tried that and it didn't work for me at first. This is better than downloading the binary. Will change argocd to use `go install` as well. 👍🏻